source/components/playerlist.rst

```diff
@@ -0,0 +1,26 @@
+==================
+Player List (Tab List)
```

Comment:
redirect needed for this one too, see how the existing redirect was done in `conf.py`